Core/MaterialParameters.h

```diff
@@ -53,6 +53,7 @@ class MaterialParameters
   MaterialParameters(const std::vector<G4double> & oPsLF, const std::vector<G4double> & oPsProb, 
                       G4double pPsLF, G4double pPsFrac, 
                       const std::vector<G4double> & DirectLF, const std::vector<G4double> & DirectProb);
+
```

Comment:
Missing 
static const G4double pPsTauVaccum;
?

Reply:
are u referring to the line 59?

Reply:
Oh sorry miss that